src/api/index.tsx

```diff
@@ -266,3 +276,50 @@ export const getDiff = async ({
     query: path ? { file: path } : undefined,
   });
 };
+
+export const createOrUpdateComment = async ({
+  /* istanbul ignore next */
+  _callApi = callApi,
+  addonId,
+  apiState,
+  cannedResponseId,
+  comment,
+  commentId,
+  fileName,
+  line,
+  versionId,
+}: {
+  _callApi?: typeof callApi;
+  addonId: number;
+  apiState: ApiState;
+  cannedResponseId: number | void;
```

Comment:
Yeah, I prefer passing explicit undefined values because it lets TS tell you when you forget to pass a value (which would be a mistake). I suppose this is a different case since one needs to define one field or the other but not both. I guess that makes it a good fit for a question mark type.